### PR TITLE
distance matrix request - add traffic_model to request and durationInTraffic to response

### PR DESCRIPTION
**Pull request by ctmckenna**
google returns durationInTraffic in response to distance matrix requests, so adding that to DistanceMatrixElement.  Also adding ability to specify traffic model for distance matrix requests

### CODE REVIEW

**Comment by ctmckenna** [comment]
I signed it!

File: src/main/java/com/google/maps/DistanceMatrixApiRequest.java [hunk 1]
```diff
@@ -19,12 +19,7 @@
 
 import com.google.maps.DirectionsApi.RouteRestriction;
 import com.google.maps.DistanceMatrixApi.Response;
-import com.google.maps.model.DistanceMatrix;
-import com.google.maps.model.LatLng;
-import com.google.maps.model.TransitMode;
-import com.google.maps.model.TransitRoutingPreference;
-import com.google.maps.model.TravelMode;
-import com.google.maps.model.Unit;
+import com.google.maps.model.*;
```
Comment (domesticmouse) [inline src/main/java/com/google/maps/DistanceMatrixApiRequest.java]: Can you please revert this change? We don't used star imports, per the Google Java style guide lines.

File: src/main/java/com/google/maps/model/DistanceMatrixElement.java [hunk 2]
```diff
@@ -35,6 +35,8 @@
    */
   public Duration duration;
 
+  public Duration durationInTraffic;
```
Comment (domesticmouse) [inline src/main/java/com/google/maps/model/DistanceMatrixElement.java]: Please add Javadoc

**Comment by ctmckenna** [comment]
removed import *, and added javadoc for durationInTraffic field

**Comment by domesticmouse** [comment]
Can you please add a test that confirms that the `traffic_model` parameter is included in the outbound request, and the `durationInTraffic` is populated on the way back in. Thanks!

**Comment by ctmckenna** [comment]
I just pushed a test case.  It verifies that traffic_model was included in the outbound request by checking the params table.  I also manually checked that it was included in the get request.

**Comment by domesticmouse** [comment]
Thank you!
